gems/sorbet/lib/gem-generator-tracepoint/tracepoint_serializer.rb

```diff
@@ -215,10 +215,10 @@ def anonymous_id
 
       def gem_from_location(location)
         match =
+          location&.match(/^.*\/gems\/([^\/]+)-([^-\/]+)\//i) || # gem
           location&.match(/^.*\/(ruby)\/([\d.]+)\//) || # ruby stdlib
           location&.match(/^.*\/(jruby)-([\d.]+)\//) || # jvm ruby stdlib
-          location&.match(/^.*\/(site_ruby)\/([\d.]+)\//) || # rubygems
-          location&.match(/^.*\/gems\/(?:(?:j?ruby-)?[\d.]+(?:@[^\/]+)?(?:\/bundler)?\/)?gems\/([^\/]+)-([^-\/]+)\//i) # gem
```

Comment:
This regex was intentionally not too general.
Can we expand it a bit to handle your case(or add an additional `||` component) rather than expanding it as much?

Reply:
No problem :) In fact to compensate for the lack of testing of this function, I've made this https://regexr.com/57t2q. It'd be really nice if that kind of thing was in the gem tests, but I don't have time to do that.